docs/ros/config/yaml.mdx

```diff
@@ -6,3 +6,892 @@ toc_min_heading_level: 2
+    platform:
+      cpr-a200-0000: 192.168.131.1
+    onboard: {}
+    remote: {}
+  ros2:
+    username: administrator
+    namespace: a200_0000
+    domain_id: 0
+    rmw_implementation: rmw_fastrtps_cpp
+platform:
+  controller: ps4
+  decorations:
+    front_bumper:
+      enabled: true
+      model: default
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+      extension: 0.0
+    rear_bumper:
+      enabled: true
+      model: default
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+      extension: 0.0
+    structure:
+      enabled: true
+      model: sensor_arch_300
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+    top_plate:
+      enabled: true
+      model: pacs
+      xyz: [0.0, 0.0, 0.0]
+      rpy: [0.0, 0.0, 0.0]
+  extras:
+    urdf: null
+    control: null
+accessories:
+  box:
+  - name: user_bay_cover
+    parent: top_plate_link
+    xyz: [0.0, 0.0, 0.00735]
+    rpy: [0.0, 0.0, 0.0]
+    size: [0.4, 0.4, 0.002]
+  cylinder: []
+  link: []
+  mesh: []
+  sphere: []
+mounts:
+  bracket:
+  - parent: top_plate_mount_d1
+    xyz: [0.0, 0.0, 0.0]
+    rpy: [0.0, 0.0, 0.0]
+    model: horizontal
+  fath_pivot:
+  - parent: sensor_arch_mount
+    xyz: [0.0, 0.0, -0.021]
+    rpy: [3.1415, 0.0, 0.0]
+    angle: 0.0
+  riser: []
+sensors:
+  camera:
+  - model: intel_realsense
+    urdf_enabled: true
+    launch_enabled: true
+    parent: fath_pivot_0_mount
+    xyz: [0.0, 0.0, 0.0]
+    rpy: [0.0, 0.0, 0.0]
+    ros_parameters:
+      camera_name: camera_0
+      device_type: d435
+      serial_no: '0'
+      enable_color: true
+      rgb_camera.profile: 640,480,30
+      enable_depth: true
+      depth_module.profile: 640,480,30
+      pointcloud.enable: true
+  gps: []
+  imu: []
+  lidar2d:
+  - model: hokuyo_ust10
+    urdf_enabled: true
+    launch_enabled: true
+    parent: bracket_0_mount
+    xyz: [0.0, 0.0, 0.0]
+    rpy: [0.0, 0.0, 0.0]
+    ros_parameters:
+      laser_frame_id: lidar2d_0_laser
+      ip_address: 192.168.131.20
+      ip_port: 10940
+      angle_min: -3.141592653589793
+      angle_max: 3.141592653589793
+  lidar3d:
+  - model: velodyne_lidar
+    urdf_enabled: true
+    launch_enabled: true
+    parent: sensor_arch_mount
+    xyz: [0.0, 0.0, 0.0]
+    rpy: [0.0, 0.0, 0.0]
+    ros_parameters:
+      frame_id: lidar3d_0_laser
+      device_ip: 192.168.131.25
+      port: 2368
+      model: VLP16
+      fixed_frame: lidar3d_0_laser
+      target_frame: lidar3d_0_laser
+```
+</p>
+</details>
+
+## Serial Number
+The Clearpath serial number is composed of two sections: an alpha-numerical code corresponding to the robot platform, followed by an integer corresponding to the unit number, and separated by a hyphen, `-`. e.g. `a200-0001`.
+
+At this moment, the supported robot platforms are:
+1. [Husky A200](/docs/robots/outdoor_robots/husky/user_manual_husky/#introduction): `a200-0000`
+2. [Jackal J100](/docs/robots/outdoor_robots/jackal/user_manual_jackal/#introduction): `j100-0000`
```

Comment:
These need to be relative links so that they always navigate to the same version that is currently being viewed.

Reply:
Done, changed to relative